clojure-ts-mode.el

```diff
@@ -909,7 +915,7 @@ and (:defn) is converted to (:inner 1)."
      ((equal spec :defn) (list :inner current-depth))
      (t nil))))
 
-(defun clojure-ts--dynamic-indent-for-symbol (symbol-name)
+(defun clojure-ts--dynamic-indent-for-symbol (symbol-name symbol-namespace)
```

Comment:
I think most of the code in `clojure-mode` and CIDER uses `sym` and `ns` for those two - short, yet fairly descriptive names.

Reply:
Also - looking at the code, you made this optional, but you didn't mark it as `&optional`.

Reply:
Fixed both. Thanks!